lib/plugins/create/templates/plugin/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// https://github.com/serverless/serverless/blob/master/docs/04-extending-serverless/01-creating-plugins.md
+class ServerlessPlugin {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+
+    this.commands = {
+      deploy: {
+        lifecycleEvents: [
+          'functions',
```

Comment:
maybe two events, `hello` and `world`, so we can show people how they can create multiple events